lib/AppInfo/Application.php

```diff
@@ -38,6 +40,7 @@
 use OCA\Mail\Service\MailTransmission;
 use OCA\Mail\Service\UserPreferenceSevice;
 use OCP\AppFramework\App;
+use OCP\EventDispatcher\IEventDispatcher;
```

Comment:
Fine by me but then this becomes 17 only

Reply:
Good point. 0.17.0 seems like a good last release for 16. Will increase the min version in another PR.